app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/background"
+        android:layout_width="fill_parent"
+        android:layout_height="180dp"
+        android:scaleType="centerCrop"
+        android:src="@drawable/login_cover"
+        app:layout_constraintTop_toTopOf="parent"
```

Comment:
It would be okey to order the tags like first the `android:`, then the `app:` and last the `tools:`. But you don't need to order by yourself since there is a shortcut that make it for you. I don't remember how it is in Linux, but I think that maybe it is `Ctrl+Alt+L`. If it is not, talk to me and I help you.